nixos/modules/services/networking/networkmanager.nix

```diff
@@ -177,7 +177,7 @@ in {
       basePackages = mkOption {
         type = types.attrsOf types.package;
         default = { inherit (pkgs)
-                            networkmanager modemmanager wpa_supplicant
+                            networkmanager modemmanager wpa_supplicant crda
```

Comment:
You're going to need to do this wherever this option is used. See GNOME3, Pantheon etc.

Reply:
Done.